dashboard/test/models/user_test.rb

```diff
@@ -875,67 +875,17 @@ class UserTest < ActiveSupport::TestCase
   test 'user should prefer working on 20hour instead of hoc' do
```

Comment:
This isn't actually explicit in our codebase, rather it is an artifact of the order in which `user_scripts` are returned.